packages/helm-charts/ethstats/templates/ethstats.ingress.yaml

```diff
@@ -10,6 +10,8 @@ metadata:
     component: ethstats
   annotations:
     kubernetes.io/tls-acme: "true"
+    nginx.ingress.kubernetes.io/ssl-redirect: "false"
+    nginx.ingress.kubernetes.io/permanent-redirect: "https://{{ .Release.Namespace }}-celostats.celo-networks-dev.org"
```

Comment:
Should this be `{{ .Values.domain.name }}`?

Reply:
thanks!